public/pages/CreateMonitor/containers/CreateMonitor/utils/constants.js

```diff
@@ -1,4 +1,5 @@
 import { OPERATORS_MAP } from '../../../components/MonitorExpressions/expressions/utils/constants';
```

Comment:
The changes in this file are all from https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/pull/77